addons/mail/static/src/components/attachment_list/attachment_list.xml

```diff
@@ -21,6 +21,15 @@
                     />
                 </t>
             </div>
+            <div class="o_AttachmentList_partialList o_AttachmentList_preview">
+                <t t-foreach="attachmentList.attachmentLinkPreviews" t-as="attachment" t-key="attachment.localId">
+                    <AttachmentLinkPreview
+                        class="o_AttachmentList_attachment o_AttachmentList_preview"
+                        attachmentLinkPreviewLocalId="attachment.localId"
+                        isCompact="props.isCompact"
```

Comment:
no props, I guess I just need to finish my message view PR first to solve these issues